### DISCUSSION TITLE
Build cake

### DISCUSSION THREAD

**Discussion by Gani-23** [Mapsui/Mapsui#1852] [Q&A]
Hi Morning, 
I was wondering if you create cake.build just to automate the process of clean, test, publish and deploy automatically by depending on the os

**Comment by pauldendulk** [comment]
We do not use cake.build. It seems an interesting project to me but I wonder if we could use it because we have to deal with some nasty things that might not work well with cake. We support many platforms, and many of those have quirks that we need to work around. 

What were you suggesting? Do you think cake.build could be a good idea?
